Dockerfile.mysql

```diff
@@ -0,0 +1,14 @@
+# Используем официальный образ MySQL
+FROM mysql:8.0
+
+# Установка ENV переменных для инициализации базы данных
+ENV MYSQL_DATABASE=app_db MYSQL_USER=app_user MYSQL_PASSWORD=1234 MYSQL_ROOT_PASSWORD=rootpassword

```

Comment:
Using hardcoded passwords (MYSQL_PASSWORD and MYSQL_ROOT_PASSWORD) is a security risk. Consider using Docker secrets or environment variables passed at runtime to manage sensitive information securely.